zarf.yaml

```diff
@@ -0,0 +1,16 @@
+# yaml-language-server: $schema=https://raw.githubusercontent.com/defenseunicorns/zarf/main/zarf.schema.json
```

Comment:
should this use w/e version of zarf is `uds zarf version`?

Reply:
Solid maybe?
I definitely just copy/pasted this from somewhere... but version pinning seems like it might be useful.
Also feels like it would be easy to get out-of-date or cause some confusion based on whatever env specific version of UDS is being used 🤔 

Do we have a good renovate (or something) pattern for this anywhere?

Reply:
I also don't have a very good sense of how often the schema changes. If it's relatively infrequent maybe it's not worth worrying about since we would probably be encouraging staying up-to-date anyways..

Reply:
Could switch this to the schema from uds-cli (`https://raw.githubusercontent.com/defenseunicorns/uds-cli/main/zarf.schema.json`) and pin to a specific uds tag + have it tracked by renovate if we want to be very careful here.

Separately - in uds-core we put schemas in our .vscode instead of adding at the top of the zarf file - https://github.com/defenseunicorns/uds-core/blob/main/.vscode/settings.json - maybe good to do that instead (and adopt the applicable renovate config to track that comment)?

Reply:
I think I added the renovate piece correctly, but could use a spot check before we resolve this one 👍

Reply:
This should work - I think you technically only need [this single match](https://github.com/defenseunicorns/uds-core/blob/main/renovate.json#L69-L76) and might not even need the renovate comment piece...not sure it's being used in uds-core either.

Reply:
I ran it against a fork and it didn't seem to explode..
<img width="1198" alt="image" src="https://github.com/defenseunicorns/uds-identity-config/assets/9499393/b4baa56c-f2d0-4f82-9240-bb046689f0b9">

Reply:
Gotcha.
Good call.

The updates on the fork also didn't explode..
I'll push a simplified config to this PR.